packages/root/src/cli/build.ts

```diff
@@ -364,7 +365,20 @@ export async function build(rootProjectDir?: string, options?: BuildOptions) {
       path.join(distDir, 'server/render.js')
     );
     const renderer = new render.Renderer(rootConfig, {assetMap, elementGraph});
-    const sitemap = await renderer.getSitemap();
+    let sitemap = await renderer.getSitemap();
+
+    // If the `--filter` flag is passed, filter only the paths that match the
```

Comment:
clarity nit: `filter only the paths that match` -> `build only the paths that match`